libnet/Makefile.am.common

```diff
@@ -5,7 +5,6 @@
 # All rights reserved.
```

Comment:
Agreed. I'm not exactly sure what the *~ business is all about. To be honest, I never encountered a file of this type during builds, but whatever autoreconf made, can be deleted using git clean. Whatever configure made, is handled by `make distclean`. Everything else is handled by `make clean`. 
There's no reason to ask to clean specific filetypes if they were created by any of the buildtools. At least not nowadays